### PR TITLE
initial implementation of the Serverless Lambda function in Scala.js

### PR DESCRIPTION
**Pull request by bpholt**

### CODE REVIEW

File: src/main/scala/com/dwolla/aws/lambda/Handler.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  @JSExportTopLevel("removeCloudWatchAlarms")
+  val handler: ScheduledHandler = (event, _, callback: LambdaCallback[Unit]) ⇒ {
+    Stream.emit(event)
+      .covary[IO]
+      .through(handleScheduledEvents[IO])
+      .compile
+      .drain
+      .unsafeRunAsync {
+        case Left(t) ⇒ callback(js.Error(t.getMessage), js.undefined)
+        case Right(()) ⇒ callback(null, ())
```
Comment (lochnesh) [inline src/main/scala/com/dwolla/aws/lambda/Handler.scala]: I think the answer is no, but want to confirm anyway.  Do we have to respond to the AutoScalingGroup after receiving this event?

File: src/main/scala/com/dwolla/aws/lambda/Handler.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      }
+  }
+
+  def handleScheduledEvents[F[_]](input: Stream[F, ScheduledEvent])
```
Comment (lochnesh) [inline src/main/scala/com/dwolla/aws/lambda/Handler.scala]: I noticed you are calling this a `ScheduledEvent` could you please explain why?  Does it have something to do with the stream processing, NodeJS, or the Autoscaling event the lambda is receiving?

Comment (bpholt) [reply src/main/scala/com/dwolla/aws/lambda/Handler.scala]: That's what AWS calls this event type. The types here are derived from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/50dafbd8e340f232392f5d7b88d163a046ebfdd5/types/aws-lambda/index.d.ts#L325-L337

File: src/main/scala/com/dwolla/aws/lambda/Handler.scala (hunk 1 above)
Comment (bpholt) [reply src/main/scala/com/dwolla/aws/lambda/Handler.scala]: No, this event is just informational, not part of a workflow.

File: src/main/scala/com/dwolla/aws/lambda/Handler.scala (hunk 2 above)
Comment (bpholt) [reply src/main/scala/com/dwolla/aws/lambda/Handler.scala]: [Here it is](https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ScheduledEvent.java#L25) in the Java Lambda library, too.

**Pull request merged**